app/src/main/java/com/yterletskyi/happyfriend/features/pin/ui/PinFragment.kt

```diff
@@ -24,5 +25,9 @@ class PinFragment : BaseBindingFragment<FragmentPinBinding>(
         viewModel.pinProgressLiveData.observe(viewLifecycleOwner) { progress ->
             binding.pinProgressView.step = progress
         }
+
+        with(binding.pinTitle) {
+            text = getString(R.string.pin_enter_title)
```

Comment:
can we set it via xml with just one line?

Reply:
but if we will want to change text from Enter a PIN to Repeat your PIN, we have to do that from code

Reply:
oh make sense -- don't change it then